clientdb/order.go

```diff
@@ -669,11 +677,13 @@ func DeserializeOrder(nonce order.Nonce, r io.Reader) (
 // non-default values of the additional data will be set on the given order.
 func deserializeOrderTlvData(r io.Reader, o order.Order) error {
 	var (
-		selfChanBalance   uint64
-		sidecarTicket     []byte
-		channelType       uint8
-		allowedNodeIDs    []byte
-		notAllowedNodeIDs []byte
+		selfChanBalance            uint64
+		sidecarTicket              []byte
+		channelType                uint8
+		allowedNodeIDs             []byte
+		notAllowedNodeIDs          []byte
+		bidUnannoucedChannels      uint8
```

Comment:
we cannot store `bool`

Reply:
Yeah makes sense, oh all those wasted bits...